ci/launch_cluster.sh

```diff
@@ -45,9 +45,17 @@ aws_region: us-west-2
 os_name: cent-os-7-dcos-prereqs
 key_helper: true
 instance_type: m4.large
-num_public_agents: 1
-num_private_agents: 3
 num_masters: 3
+ssh_user: centos
```

Comment:
Is this really required?

Reply:
I think I added it for ensuring that we have a consistent SSH user, regardless of the installation method or target OS. But I can remove it.